### PR TITLE
Run internet chapter through hemingway

### PR DESCRIPTION
**Pull request by Videocard**
Fixed grammar and wording using Hemingway app to highlight hard-to-read areas.

:cake:

### CODE REVIEW

File: en/how_the_internet_works/README.md [hunk 1]
```diff
@@ -31,17 +31,17 @@ It looks like this:
...
 
-When you send a letter, it needs to have certain features to be delivered correctly: an address, stamp etc. You also use a language that the receiver understands, right? The same is with *data packets* you send in order to see a website: you use a protocol called HTTP (Hypertext Transfer Protocol).
+When you send a letter, it needs to have certain features to be delivered correctly: an address, stamp etc. You also use a language that the receiver understands, right? The same applies to the *data packets* you send to see a website. We use a protocol called HTTP (Hypertext Transfer Protocol).
 
-So, basically, when you have a website you need to have a *server* (machine) where it lives. The *server* is waiting for any incoming *requests* (letters that ask the server to send your website) and it sends back your website (in another letter).
+So, basically, when you have a website, you need to have a *server* (machine) where it lives. When the *server* receives an incoming *request*, (in a letter) it sends back your website (in another letter).
```
Comment (olasitarska) [inline en/how_the_internet_works/README.md]: > *request*, (in a letter)

comma after parentheses, maybe?

Comment (Videocard) [inline en/how_the_internet_works/README.md]: Good idea, I'll make the change!

**Comment by aniav** [comment]
Yay! :)
